tianshou/data/collector.py

```diff
@@ -364,6 +354,11 @@ def sample(self, batch_size: int) -> Batch:
             the buffer, otherwise it will extract the data with the given
             batch_size.
         """
+        import warnings
+        warnings.warn(
+            'Collector.sample is deprecated and will cause error if you use '
+            'prioritized experience replay! Collector.sample will be removed'
+            ' after 0.3. Use policy.update instead!', DeprecationWarning)
```

Comment:
`DeprecationWarning` doesn't print any warning in my local.

Reply:
Nice catch! This [so question](https://stackoverflow.com/a/20960427/9191338) explains the problem.

Maybe we should add the following lines in tianshou.__init__

```python
import warnings
warnings.simplefilter('always', DeprecationWarning)
```

Reply:
I change it to Warning... Also the last pr.